source/agent/conference/conference.js

```diff
@@ -108,6 +108,8 @@ var Conference = function (rpcClient, selfRpcId) {
   var rtcController;
   let quicController;
 
+  var roomToken = Math.round(Math.random() * 1000000000000000000) + '';
```

Comment:
Change random function

Reply:
use crypto.randomBytes to generate secure random bytes